core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionNetworkOutput.java

```diff
@@ -138,8 +139,8 @@ public boolean sendSync(Transmission transmission) {
             try {
                 // POST the transmission data to the endpoint
                 request = createTransmissionPostRequest(transmission);
-                httpClient.enhanceRequest(request);
-                response = httpClient.sendRequest(request);
+                LazyHttpClient.enhanceRequest(request);
+                response = httpClient.execute(request);
```

Comment:
is this why you make instance to be CloseablehttpClient instead of lazyhttpclient? sendRequest is removed, which is a wrapper method around calling execute?

Reply:
Using `CloseableHttpClient` also makes it easy to mock in tests